### ISSUE TITLE
declaration-empty-line-before

### ISSUE THREAD

**Issue by agordeev-zz** [stylelint/stylelint#2841]
8.0.0
`declaration-empty-line-before`

```css
.class {
    display: block;

    &::after { // comment
        content: '';
    }
}
```
```json
"declaration-empty-line-before": [ "always", {
    "except": [
        "after-declaration",
        "first-nested"
    ],
    "ignore": [
        "after-comment"
    ]
}],
```
```shell
Expected empty line before declaration   declaration-empty-line-before
```

The problem appeared in 8.0.0 (was 7.12.0). All errors are equal, there is a comment on the same line with selector.

**Comment by hudochenkov** [comment]
Thanks for creating this issue but we are closing it as issues need to follow our issue template, so that we can clearly understand your particular circumstances.

Please help us to help you by [recreating the issue](https://github.com/stylelint/stylelint/issues/new) using the template.

**Comment by jeddy3** [comment]
@agordeev-zz Should [be fixed](https://github.com/stylelint/stylelint/pull/2827) in the upcoming `8.1.0` release.
